src/transformers/models/autoformer/modeling_autoformer.py

```diff
@@ -1802,11 +1793,7 @@ def forward(
                 attentions=encoder_outputs[2] if len(encoder_outputs) > 2 else None,
             )
 
-        # decoder input
         dec_input = transformer_inputs[:, self.config.context_length :, ...]
-        # mean_enc_input = enc_input.mean(dim=1).unsqueeze(1).repeat(1, dec_input.size(1), 1)
```

Comment:
same, revert this delete after Informer merge

Reply:
done in https://github.com/huggingface/transformers/pull/21891/commits/bbb722c0b3ebe8be4b3725e206d8b1bd11bee312